@ecomplus/storefront-components/src/js/TheProduct.js

```diff
@@ -492,14 +509,57 @@ export default {
 
   created () {
     if (this.product) {
-      this.body = this.product
+      this.body = { ...this.product }
```

Comment:
```suggestion
      this.body = this.product
```

Por que alterou isso? 🤔 
Copiar objetos é lento, não tendo motivo pra fazer o melhor sempre é não fazer ..

Reply:
Ok